wiki/Ranking_Criteria/osu!mania/id.md

```diff
@@ -0,0 +1,199 @@
+
+### Peraturan
+
+- **Mod `Auto` harus mencapai skor maksimal (1,000,000) di semua tingkat kesulitan.**
+- **Dua note tidak boleh ditempatkan dalam satu kolom pada timestamp yang sama.**
+- **Tidak boleh ada kolom yang dikosongi.** Jika Anda ingin menggunakan key yang lebih sedikit, ubah jumlah key yang terdapat pada Song Setup.
+- **Tidak boleh ada lebih dari 6 note yang ditekan pada saat yang bersamaan di tingkat kesulitan Insane atau dibawahnya.** Menggunakan lebih dari 6 note juga harus mengikuti spread yang wajar ke tingkat kesulitan berikutnya yang lebih rendah. Aturan ini tidak berlaku untuk bagian akhir dari long note, karena long note dilepas, tidak ditekan.
+- **Beatmap hanya dapat menggunakan 4 sampai 9 key.** Selain itu tidak didukung untuk Ranked Section.
+- **Jika terdapat beberapa key mode dalam satu set beatmap, key mode tersebut harus dicantumkan pada setiap tingkat kesulitan. Jika tidak, key mode tidak perlu dicantumkan.**
+
+### Pedoman
+
+- **Setiap note harus berkorelasi dengan suara yang ada dalam musik.** Hal ini harus menjadi suara yang berbeda, namun juga secara koheren mewakili suara berkelanjutan awal atau akhir yang tidak dapat dibedakan. Pengecualian untuk penyederhanaan dapat dilakukan saat mengikuti music dengan sangat akurat...
+  - akan menjadi tidak masuk akal untuk bermain dengan akurasi penuh.
+  - akan secara efektif jika ditampilkan secara konsisten, dan dapat diprediksi.
+  - akan menyebabkan lonjakan kesulitan yang kurang tepat.
+- **Hindari menggunakan sampel hitsound storyboard dengan volume yang mirip dengan hitsound yang terdapat pada note.** Karena storyboard hitsound menyebabkan distorsi umpan balik audio dari objek di beatmap, meletakkan hitsound pada note harus di prioritaskan. Namun, sampel storyboard dapat digunakan saat keysound di ekstrasi langsung dari lagu dan saat tingkat kesulitan rendah memiliki note yang sedikit untuk mendukung hitsound yang diperlukan.
+- **Jangan gunakan keysound tanpa dukungan dari hitnormal.** Jika hitsound terlalu menyatu dengan lagu, umpan balik saat menekan note akan menjadi sangat minim. Namun, ketika keysound di ekstrak langsung dari sebuah lagu dan digunakan sebagai pengganti audio lagu, keysound tidak perlu dukungan dari hitnormal.
+- **Hindari menggunakan hitsound yang sama untuk beberapa note dalam satu chord.** Hal ini menyebabkan umpan balik menjadi lebih keras dari yang dibutuhkan. Ini juga berlaku untuk hitsound aktif dan storyboard hitsound yang digunakan pada timestamp yang sama.
+- **Perubahan waktu harus harus memiliki scroll speed terpadu dengan menyesuaikan slider velocity.** Hal ini dilakukan dengan menetralkan scroll speed dari semua timing ke bagian yang paling sering digunakan di beatmap. Nomor ditampilkan dalam tanda kurung setelah rentang BPM dalam pemilihan lagu, misal `BPM: 60-160 (130)`. Perhatikan bahwa pedoman ini tidak memengaruhi kemampuan untuk menambahkan perubahan slider velocity yang digunakan untuk membuat efek SV.
```

Comment:
quite unsure am i should to commit this suggest or not , RC : "Note that this guideline does not affect the ability to add slider velocity changes which are used to create SV effects." so i think i should use "SV" instead of slider velocity here. I'll keep this one for now

Reply:
it should help people to get familiar with the terms better, especially for newcomers, perhaps "efek slider velocity (SV)" will do?